src/overlays/actors/ovl_Obj_Mure/z_obj_mure.h

```diff
@@ -7,6 +7,20 @@ struct ObjMure;
 
 typedef void (*ObjMureActionFunc)(struct ObjMure*, PlayState*);
 
+#define OBJ_MURE_GET_CHNUM(thisx) (((thisx)->params >> 12) & 0xF)
+#define OBJ_MURE_GET_PTN(thisx) (((thisx)->params >> 8) & 0x7)
```

Comment:
Uhh. these might be abbreviated too far, what are they supposed to be?

Reply:
Yeah, I had the same thought when this actor was PR'd. Apparently it's from OoT debug strings, we don't know what they stand for:
https://github.com/zeldaret/mm/pull/364#discussion_r731471642